client/client.go

```diff
@@ -348,14 +349,38 @@ func loadServerTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
 	return ctx, err
 }
 
+func loadPrivateKey(conf *Config) (openssl.PrivateKey, error) {
+	var key openssl.PrivateKey
```

Comment:
Suggestion:
Might as well use the return initializer:
`(key openssl.PrivateKey, err error)`

Reply:
discussed privately, decided to keep it in.